developer_manual/design/navigation.rst

```diff
@@ -0,0 +1,153 @@
+.. sectionauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+.. codeauthor:: John Molakvoæ <skjnldsv@protonmail.com>
+..  _navigation:
+
+===============
+App Navigation
+===============
+
+Introduction
+=============
+
+The main navigation menu represent the most important part of your app.
```

Comment:
represent**s**

Reply:
But also, I wouldn’t say »most important part of your app« since really the most important part is the content. ;) Maybe more something like »It is the main navigation of your app, so it needs to be simple, organized and responsive.«?